get_webpack_manifest.py

```diff
@@ -0,0 +1,23 @@
+import json
+import os
+import warnings
+from django.conf import settings
+
+
+def get_webpack_manifest(path=None):
+    manifest = {}
+
+    if not path:
+        path = os.path.join('webpack/_build/manifest.json')
```

Comment:
`os.path.join` is doing nothing here
```suggestion
        path = 'webpack/_build/manifest.json'
```